apps/laboratory/playwright.config.ts

```diff
@@ -21,7 +21,7 @@ export default defineConfig<ModalFixture>({
   expect: {
     timeout: getValue(60, 15) * 1000
   },
-  timeout: 60 * 1000,
+  timeout: 90 * 1000,
```

Comment:
Why do we need increase this? For which situations 60 was not enough? Might it be something we need to improve?

Reply:
Yeah i increased the test timeout to not cause conflicts with other tests, since i added a test which takes 30+ seconds.